rest-service/manager_rest/config.py

```diff
@@ -184,11 +184,11 @@ def load_from_db(self):
             setattr(self, conf_value.name, conf_value.value)
 
         stored_brokers = session.query(models.RabbitMQBroker).all()
-        broker_hosts = [broker.host for broker in stored_brokers]
+        self.amqp_host = [b.host for b in stored_brokers]
+        self.amqp_management_host = [b.management_host for b in stored_brokers]
```

Comment:
this is now a list too, and the client copes with both a list and a single value,

https://github.com/cloudify-cosmo/cloudify-common/blob/e980d9222160c3c8562f497fe97e0045b70c4d7c/cloudify/rabbitmq_client.py#L32-L33

Reply:
I've verified that all usages of AMQPClient inside of cloudify-manager do create a new client per request, so we're OK wrt. all the `.pop()` calls that the client does